app/javascript/components/components/Header.js

```diff
@@ -30,6 +30,11 @@ class Header extends Component {
               <a href="/countries" className="nav-link" style={{ backgroundColor: "#448899", color: "white"}}>Countries</a>
             </NavItem>
           }
+          {logged_in &&
+            <NavItem>
+              <a href="/countries" className="nav-link" style={{ backgroundColor: "#448899", color: "white"}}>Countries</a>
+            </NavItem>
+          }
```

Comment:
It might be worth extracting these styles into the stylesheet. Again not a deal breaker, but repeating code is a bummer.